wgsl/index.bs

```diff
@@ -110,6 +110,9 @@ div.syntax > p > a > code {
   font-style: normal;
   font-weight: bold;
 }
+.hidden {
```

Comment:
This is used to hide the _disambiguate_template tokens in the grammar.
It's an implementation detail for the Treesitter parser, and so it should *not* be visible in the rendered spec.